src/foam/dao/SinkDAO.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @license
+ * Copyright 2022 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.dao',
+  name: 'SinkDAO',
+  extends: 'foam.dao.ProxyDAO',
+  flags: [ 'js '],
+
+  documentation: 'DAO which wraps Sink on Select.  Useful for client logic which does not directly have access to the select() and cannot call await itself, such as ChoiceView.',
```

Comment:
I can't understand what this is for based on this documentation.

Reply:
removed in predicate rework.